docs/tools/offline/cli-setup.md

```diff
@@ -155,7 +156,8 @@ The `-G` switch tells Mbed CLI to set this as a global setting, allowing all pro
 
 Mbed CLI supports a setting for each toolchain path. Below is a list of these settings, along with a description of what path is expected in each setting.
 
-- `ARM_PATH`: The path to the *base* directory of your Arm Compiler installation. This should be the directory containing the directory containing the binaries for `armcc` and friends. For example, if your Arm Compiler 5 executable `armcc` is located at `/home/redacted/ARM_Compiler_5.06u5/bin/armcc`, you set `ARM_PATH` to `/home/redacted/ARM_Compiler_5.06u5`
+- `ARM_PATH`: The path to the *base* directory of your Arm Compiler 5 installation. This should be the directory containing the directory containing the binaries for `armcc` and friends. For example, if your Arm Compiler 5 executable `armcc` is located at `/home/redacted/ARM_Compiler_5.06u5/bin/armcc`, you set `ARM_PATH` to `/home/redacted/ARM_Compiler_5.06u5`
+- `ARMC6_PATH`: The path to the *binary* directory of your Arm Compiler 6 installation. This should be the directory containing the binaries for `armar` and friends. For example, if your Arm Compiler 6 executable `armar` is located at `C:/Program Files/ARM/armcc6.8/bin/armar`, you set `ARMC6_PATH` to `C:/Program Files/ARM/armcc6.8/bin`
```

Comment:
Technically, the tools look for `armclang`, but they're always in the same folder.